specs/cm/change-mgt-vocab.ttl

```diff
@@ -28,10 +28,15 @@
 oslc_cm:
   a owl:Ontology ;
   dcterms:title "The OSLC Change Management(CM) Vocabulary" ;
+  rdfs:label "The OSLC Change Management(CM) Vocabulary" ;
```

Comment:
I would suggest dropping the The article and adding a space before the opening bracket here as well.

Reply:
Done